charts/backend/templates/secrets.yaml

```
@@ -18,6 +18,10 @@ stringData:
   KC_DB_PASSWORD: {{ .Values.secrets.postgres.dbPassword }}
   KC_DB_URL_HOST: {{ .Values.global.opentdf.common.postgres.host }}
   KC_DB_URL_DATABASE: {{ .Values.keycloak.externalDatabase.database }}
+  KC_HOSTNAME_STRICT_HTTPS: {{ .Values.keycloak.hostname.strictHttps | quote }}
+  KC_HOSTNAME_STRICT: {{ .Values.keycloak.hostname.strict | quote }}
+  KC_HTTP_ENABLED: {{ .Values.keycloak.http.enabled | quote }}
+  CLAIMS_URL: {{ .Values.keycloak.claims.url }}
```

If you are going to move it here, why not:

```suggestion
  {{- with index .Subcharts "entitlement-pdp" }}
  CLAIMS_URL: {{ printf "http://%s:%v" (include "entitlement-pdp.fullname" .) .Values.config.listenPort }}
  {{- end }}
```